runtime/plugin/tohtml.lua

```diff
@@ -0,0 +1,770 @@
+--- Neovim plugin for converting a syntax highlighted file to HTML.
+
+if vim.g.loaded_2html_plugin then
```

Comment:
Do we use this pattern in Lua plugins? Most of the code should live in a module that is not loaded until `require()`.

Reply:
Yeah, but for ease of convenience, I'll only move the functions to the right files when I'm done with the features.

Reply:
Should live somewhere in `/lua` (which is the "`autoload` at home" for Lua functions).  There's really no need for this to be in `/plugin` and sourced on every startup).

Reply:
Okay, I moved the functions to lua/tohtml.lua

Reply:
Final location still TBD; we don't like having a lot of unrelated top-level scripts ;)

But this is much closer to how it'll have to be in the end.